stubs/psutil/psutil/__init__.pyi

```diff
@@ -198,6 +198,11 @@ class Process:
     if sys.platform != "darwin":
         def io_counters(self) -> pio: ...
         def ionice(self, ioclass: int | None = None, value: int | None = None) -> pionice: ...
+        @overload
+        def cpu_affinity(self, cpus: None) -> list[int]: ...
+        @overload
+        def cpu_affinity(self, cpus: list[int]) -> None: ...
+        @overload
         def cpu_affinity(self, cpus: list[int] | None = None) -> list[int] | None: ...
```

Comment:
I think we can get rid of the 3rd overload. The first two cover both cases
```suggestion
```

Reply:
Probably true for the vast majority of cases, yes; but there are valid reasons to keep the combination, for example if someone is proxying the Process object calls. I defer to you; if you want I can add the removal as a separate commit here and see if there is any impact in mypy_primer.

Reply:
Oh, now I see, thanks. Let's revert it :)

It's just that for the future, force commits are not [considered](https://github.com/python/typeshed/blob/main/CONTRIBUTING.md#submitting-changes) best practice here.

Reply:
Sorry for the force push. I've added a commit with a default value to remaining overload without, keeping the 3rd overload out, to see what happens.